src/Command/BasicCommands.php

```diff
@@ -137,27 +137,30 @@ public function xdebugStop()
    *
    * @param String $alias Drush alias of the site we want the db from.
    */
-  public function siteGrabDatabase($alias)
-  {
-    // Tables for which we want the structure and not the data.
-    $structure_tables_common = array(
-      'accesslog', 'batch', 'cache', 'cache_*', '*_cache', 'ctools_views_cache',
-      'ctools_object_cache', 'flood', 'history', 'queue', 'search_*',
-      'semaphore', 'sessions', 'watchdog'
-    );
-    $structure_tables_devel = array('devel_queries', 'devel_times');
-    $structure_tables_webform = array('webform_submitted_data');
-    $structure_tables = array_merge(
-      $structure_tables_common,
-      $structure_tables_devel,
-      $structure_tables_webform
-    );
-    $structure_tables_list = implode(', ', $structure_tables);
-    // Tables to be completely skipped.
-    $skip_tables = array('dkan_datastore_*');
-    $skip_tables_list = implode(', ', $skip_tables);
+    public function siteGrabDatabase($alias)
+    {
+      // Tables for which we want the structure and not the data.
+        $structure_tables_common = array(
+        'accesslog', 'batch', 'cache', 'cache_*', '*_cache', 'ctools_views_cache',
+        'ctools_object_cache', 'flood', 'history', 'queue', 'search_*',
+        'semaphore', 'sessions', 'watchdog'
+        );
+        $structure_tables_devel = array('devel_queries', 'devel_times');
+        $structure_tables_webform = array('webform_submitted_data');
+        $structure_tables = array_merge(
+            $structure_tables_common,
+            $structure_tables_devel,
+            $structure_tables_webform
+        );
+        $structure_tables_list = implode(', ', $structure_tables);
+      // Tables to be completely skipped.
+        $skip_tables = array('dkan_datastore_*');
+        $skip_tables_list = implode(', ', $skip_tables);
 
-    $command = "drush $alias sql-dump --structure-tables-list='" . $structure_tables_list . "' --skip-tables-list='" . $skip_tables_list . "' > excluded_tables.sql";
-    return $this->taskExec($command)->run();
-  }
+        return $this->taskExec("drush $alias sql-dump")
+            ->option('structure-tables-list', $structure_tables_list)
+            ->option('skip-tables-list', $skip_tables_list)
+            ->rawArg('> excluded_tables.sql')
```

Comment:
I am slightly concerned this won't work anymore. Too bad we don't have full test coverage!